components/menu/index.css

```diff
@@ -151,33 +79,47 @@
 	--spectrum-menu-item-min-height: var(--spectrum-component-height-100);
 	--spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-100);
 	--spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-100);
+
+	--spectrum-menu-item-linkout-icon-height: var(--spectrum-link-out-icon-size-100);
+	--spectrum-menu-item-linkout-icon-width: var(--spectrum-link-out-icon-size-100);
+
 	--spectrum-menu-item-label-font-size: var(--spectrum-font-size-100);
 	--spectrum-menu-item-label-text-to-visual: var(--spectrum-text-to-visual-100);
 
+	--spectrum-menu-item-top-to-action: var(--spectrum-spacing-50);
+	--spectrum-menu-item-top-to-checkbox: var(--spectrum-spacing-50);
 	--spectrum-menu-item-label-inline-edge-to-content: var(--spectrum-component-edge-to-text-100);
 	--spectrum-menu-item-top-edge-to-text: var(--spectrum-component-top-to-text-100);
 	--spectrum-menu-item-bottom-edge-to-text: var(--spectrum-component-bottom-to-text-100);
-
 	--spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-100);
+	--spectrum-menu-item-selectable-edge-to-text-not-selected: var(--spectrum-menu-item-selectable-edge-to-text-not-selected-medium);
 
 	--spectrum-menu-item-description-font-size: var(--spectrum-font-size-75);
 
-	--spectrum-menu-section-header-font-size: var(--spectrum-font-size-100);
-	--spectrum-menu-section-header-min-width: var(--spectrum-component-height-100);
-
-	--spectrum-menu-item-selectable-edge-to-text-not-selected: var(--spectrum-menu-item-selectable-edge-to-text-not-selected-medium);
+	--spectrum-menu-item-corner-radius: var(--spectrum-corner-radius-500);
 
 	--spectrum-menu-item-checkmark-height: var(--spectrum-menu-item-checkmark-height-medium);
 	--spectrum-menu-item-checkmark-width: var(--spectrum-menu-item-checkmark-width-medium);
```

Comment:
It looks like `--spectrum-menu-item-checkmark-height-medium`, `--spectrum-menu-item-checkmark-width-medium`, and their associated sizes (small, large, etc.) are tokens, but it doesn't look like they're being used anymore; instead the spec says to use `checkmark-icon-size-75` (and other sized friends like -100, -200, etc.).

I noticed that the checkmark is disappearing for the XL size and I'm wondering if that has something to do with it:
![Screenshot 2025-05-09 at 11 17 55 AM](https://github.com/user-attachments/assets/c7246ac7-142a-452d-b971-0c6dd8969d22)